docs/libertyDev.md

```diff
@@ -196,14 +196,36 @@ When dev mode runs with container support, it builds a container image and runs
 
 This task requires applications to be installed as loose applications. Information on configuring loose applications can be found in the [deploy task parameter documentation](deploy.md#Parameters) and the [Liberty server configuration](libertyExtensions.md#liberty-server-configuration).
 
+##### Copying Generated Configurations to Docker Image
```

Comment:
I didn't catch this for LMP, but the image is not necessarily a docker image is it? Should we just say container imagae? and then instead of dockerfile use containerfile?